package.json

```diff
@@ -19,11 +19,12 @@
     "async": "0.9.0",
     "babyparse": "0.2.1",
     "commander": "2.5.0",
+    "common": "0.2.5",
     "lodash": "2.4.1",
     "moment-timezone": "0.2.4",
     "react": "0.12.0",
     "sundial": "1.0.0",
-    "tidepool-platform-client": "0.14.0"
+    "tidepool-platform-client": "git://github.com/tidepool-org/platform-client.git#ian/uploadGroup"
```

Comment:
@ianjorgensen we obvioulsy need to change this but added while i was trying to test